docs/why.qmd

```diff
@@ -2,24 +2,224 @@
 title: "Why Ibis?"
 ---
 
+Ibis defines a Python dataframe API that supports any query engine -- the
+frontend for any backend data platform. This includes 20+ backends today, most
+SQL-based.  This allows Ibis to have excellent performance -- as good as the
+backend it is connected to -- with a consistent user experience.
+
+## What is Ibis?
+
 Ibis is the portable Python dataframe library.
```

Comment:
```suggestion
Ibis is the portable Python dataframe library meaning it's write-once, run anywhere.
```

Reply:
we want to avoid this in part because of bad connotations (Write once, debug everywhere): https://en.wikipedia.org/wiki/Write_once,_run_anywhere

Reply:
nice catch, thanks